build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.0.0-alpha1'
```

Comment:
Gradle 2.4 is fine and suits the needs of this project. We will update it when we feel it is necessary. Thanks for your contributions.